driver-core/src/test/java/com/datastax/driver/core/TestUtils.java

```diff
@@ -838,7 +811,7 @@ public static long getFreeMemoryMB() {
 
     /**
      * Helper for generating a DynamicCompositeType {@link ByteBuffer} from the given parameters.
-     *
+     * <p>
```

Comment:
Ah, good catch, IntelliJ's autoformatter behaves differently between when you have JDK 8 configured and otherwise.  For some reason with JDK 8 it omits `/`.  I'll fix it.